### ISSUE TITLE
Invalid concept hierarchy in code lists

### ISSUE THREAD

**Issue by jennet** [Swirrl/ons-data-export#48]
There are some code lists that are returning `skos:broader` concepts that do not have information associated with them. This will cause the concept tree to error.

The code lists affected on cogs-staging are:
```
http://gss-data.org.uk/def/concept-scheme/pink-book-services
http://gss-data.org.uk/def/concept-scheme/itis-services
http://gss-data.org.uk/def/concept-scheme/ons-partner-geography
http://gss-data.org.uk/def/concept-scheme/product-department
```

Full list of concepts with dead end broader concepts can be found using:
```
PREFIX skos:    <http://www.w3.org/2004/02/skos/core#>

SELECT ?scheme ?concept ?broader  {
    ?concept a skos:Concept .
    ?concept skos:inScheme ?scheme .
    ?concept skos:broader ?broader .
    FILTER NOT EXISTS {
        ?broader a skos:Concept 
    } 
}
```

**Comment by jennet** [comment]
I will remove the "dead ends" in the staging data for now, but this is another source data fix required.

**Comment by jennet** [comment]
data fix applied to #50

**Comment by jennet** [comment]
No fixes were needing in the latest data updates. Closing this issue and making a note of it on the cogs-issue https://github.com/Swirrl/cogs-issues/issues/13
